src/app/views/App.tsx

```diff
@@ -165,64 +172,86 @@ class App extends Component<IAppProps, IAppState> {
     });
   };
 
+  public toggleSidebar(): void {
+    const showSidebar = !this.state.showSidebar;
+    this.setState({ showSidebar });
+  }
+
+  public displayToggleButton = (mediaQueryList: any) => {
+    const showToggle = mediaQueryList.matches;
+    let showSidebar = true;
+    if (showToggle) {
+      showSidebar = false;
+    }
+    this.setState({ showToggle, showSidebar  });
+  }
+
   public render() {
     const classes = classNames(this.props);
     const { graphExplorerMode, error, actions, sampleQuery }: any = this.props;
+    const { showSidebar, showToggle } = this.state;
     const layout =
       graphExplorerMode === Mode.TryIt
-        ? 'col-sm-12'
-        : 'col-sm-12 col-lg-9';
+        ? 'col-sm-12 col-xs-12'
+        : 'col-xs-12 col-sm-12 col-lg-9 col-md-8';
```

Comment:
What is the reason behind using 8 columns for medium screen sizes?

Reply:
I noticed that for tablets to render properly, I'd need to use the col-md-8 and col-md-4 combination